crates/ruff_python_ast/src/context.rs

```diff
@@ -251,6 +251,11 @@ impl<'a> Context<'a> {
         self.exprs.iter().rev().nth(2)
     }
 
+    /// Return an [`Iterator`] over the current `Expr` parents.
+    pub fn iter_exprs(&self) -> impl Iterator<Item = &RefEquality<'a, Expr>> {
```

Comment:
Nit: Call it `expr_ancestors` to clarify that it doesn't iterate over the expressions but the ancestor tree?